src/Particle.js

```diff
@@ -2099,6 +2103,29 @@ class Particle {
 		});
 	}
 
+	/**
+	 * Set default auth token that will be used in each method if `auth` is not provided
+	 * @param  {String} auth A Particle access token
+	 * @returns {undefined}
+	 */
+	setDefaultAuth(auth){
+		this._defaultAuth = auth;
+	}
+	/**
+	 * Return provided token if truthy else use default auth if truthy else undefined
+	 * @param {*} auth Optional auth token or undefined
+	 * @private
+	 * @returns {String|undefined} a Particle auth token or undefined
+	 */
+	_getEffectiveAuth(auth) {
```

Comment:
minor naming nit, totally your call: how about `_getActiveAuthToken(auth)`?

Reply:
Yea, that is a better name, made change in https://github.com/particle-iot/particle-api-js/pull/145/commits/0305f4f2ae22138e4d701970a078e1de1a819657